### PR TITLE
Added link to ML samples repo

### PR DESCRIPTION
**Pull request by pkulikov**
Is it a good idea to add the link to the [ML samples repo](https://github.com/dotnet/machinelearning-samples) to the docs? If yes, I'll move it to another page in case of the tutorials index isn't perfect place (I've chosen that page because the related issue has been submitted under it).

Related to #5722

### CODE REVIEW

File: docs/machine-learning/tutorials/index.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+ms.date: 06/06/2018
 ---
 # ML.NET tutorials
 
 The following tutorials enable you to understand how to use [ML.NET](../index.md) to build custom machine learning solutions and integrate them into your .NET applications:
 
 * [Sentiment Analysis](sentiment-analysis.md): demonstrates how to apply a binary classification task using ML.NET.
 * [Taxi Fare Predictor](taxi-fare.md): demonstrates how to apply a regression task using ML.NET.
+
+For more examples that use ML.NET, check the [dotnet/machinelearning-samples GitHub repository](https://github.com/dotnet/machinelearning-samples).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/index.md]: one little nit: I usually like to keep just the repo name as the link as in `[dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples)  GitHub repository`

**Review by mairaw — CHANGES_REQUESTED** [review]
It's almost like you read my mind @pkulikov. I was going to suggest the same. 😄 Thank you for adding this. Just left a tiny comment to be addressed.

**Comment by pkulikov** [comment]
@mairaw I've addressed your comment, thank you for review.

**Review by mairaw — APPROVED** [review]
Thanks for making the changes @pkulikov!

**Pull request merged**